activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -61,7 +61,7 @@ def test_schema_dump
 
   def test_schema_dump_uses_force_cascade_on_create_table
     output = dump_table_schema "authors"
-    assert_match %r{create_table "authors", force: :cascade}, output
+    assert_match %r{create_table "authors", .*?force: :cascade}, output
```

Comment:
In the meantime I've updated the PR to make the resulting schema dump look the same for bigserial /cc @jeremy